streamz/core.py

```diff
@@ -15,6 +16,10 @@
 from tornado.locks import Condition
 from tornado.ioloop import IOLoop
 from tornado.queues import Queue
+try:
+    import confluent_kafka as ck
```

Comment:
Won't this just make for an odd error for anyone trying to use the kafka stream without confluent? Why not just import where needed in the methods?

Reply:
When I looked at the Kafka code in the sources.py file, I see that the Confluent library is imported locally.  I assumed that there must have been some desire to only make this library dependent on the Confluent library only if the user was going to communicate with Kafka.  So, I did not make it mandatory.

Reply:
Either way it wouldn't be mandatory, but in your version, you would get something like "None has no attribute..." error message when trying to use it, which would be confusing.

Reply:
Okay, I can make the import local to the to_kafka class.

Reply:
This change seems to work.

https://github.com/jsmaupin/streamz/commit/893300f55e8d016b91e9770471cc1109dd34a742

Reply:
Updated to the above comment in the latest PR.